### PR TITLE
DASH-191, DASH-323, DASH-326, DASH-327

### PR DESCRIPTION
**Pull request by guscost**
Several smaller issues from the Location Management 1.1 epic.

As part of DASH-323, this build consolidates common snippets that are re-used across the four space types. These snippets are in a new `admin-locations-snippets` module in the components folder.

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -28,6 +28,11 @@ export enum DensitySpaceFunction {
   // Note, "OTHER" is not in here since typescript does not allow "null" in enums
 }
 
+export enum DensitySizeAreaUnitTypes {
+	SQUARE_FEET = 'square_feet',
```
Comment (1egoman) [inline src/types/index.ts]: There are tabs here, fyi

File: src/components/admin-locations-snippets/index.tsx [hunk 2]
```diff
@@ -0,0 +1,234 @@
+import React, { Fragment } from 'react';
+import classnames from 'classnames';
+import comma from 'comma-number';
```
Comment (jshanley) [inline src/components/admin-locations-snippets/index.tsx]: You could also just use `Number.prototype.toLocaleString` and use `en-us`:

eg. 

``` javascript
var n = 1234
n.toLocaleString('en-us') // '1,234'
```

File: src/components/admin-locations-snippets/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+This module has lots of snippets used to render location admin pages.
+
+## AdminLocationsLeftPaneDataRow
+
+A component used to render a dataset in the left hand pane of space management list views. This
+component is a wrapper for a metric group which is displayed in one row.
+
+## AdminLocationsLeftPaneDataRowItem
+
+This component is the "child" component for each metric, and should be nested inside an AdminLocationsLeftPaneDataRow component.
```
Comment (jshanley) [inline src/components/admin-locations-snippets/README.md]: super helpful, thank you 🙌

File: src/components/admin-locations-snippets/index.tsx (hunk 2 above)
Comment (guscost) [reply src/components/admin-locations-snippets/index.tsx]: unfortunately:

```
(1234.00).toLocaleString('en-us')
"1,234"
(1234.00).toFixed(2).toLocaleString('en-us')
"1234.00"
```

Comment (jshanley) [reply src/components/admin-locations-snippets/index.tsx]: (someNumber).toFixed(2) is a String, not a Number

Comment (guscost) [reply src/components/admin-locations-snippets/index.tsx]: the point is this method is unusable for converting a number of dollars and cents to a comma-formatted string

**Pull request merged**